src/workspace/manifest.rs

```diff
@@ -160,6 +162,26 @@ impl Manifest {
         Ok(self)
     }
 
+    /// Extract optimization-passes from `[profile.release]`
+    pub fn get_profile_release_optimization_passes(&mut self) -> Option<OptimizationPasses> {
+        self.get_profile_release_table_mut()
+            .ok()?
+            .get("optimization-passes")
+            .map(|val| val.to_string())
+            .map(Into::into)
+    }
+
+    /// Extract optimization-passes from `[profile.release]`
+    #[cfg(test)]
+    pub fn set_profile_release_optimization_passes(
```

Comment:
Could just add this inline to the tests since `get_profile_release_table_mut` is public